docker/docker-compose.common.yml

```diff
@@ -0,0 +1,45 @@
+version: "3"
+
+services:
+  proxy:
+    image: dockercloud/haproxy
+    environment:
+      - BALANCE=leastconn
+      - EXTRA_GLOBAL_SETTINGS=debug
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+    ports:
+      - 80:80
+    networks:
+      - openpension
+
+  database:
+    image: postgres
+    volumes:
+      - ./database:/etc/cron.daily
+      - /var/backups:/var/backups
```

Comment:
It's true and not a good behavior, because Docker should be stateless - It' can go down and up whenever. We also should volume the actual database directory.